pkg/sql/schemachanger/scexec/scmutationexec/helpers.go

```diff
@@ -280,7 +280,6 @@ func updateColumnExprSequenceUsage(d *descpb.ColumnDescriptor) error {
 		ids.ForEach(all.Add)
 	}
 	d.UsesSequenceIds = all.Ordered()
-	d.OwnsSequenceIds = all.Ordered()
```

Comment:
one thing i'm wondering is if we need to special case IDENTITY columns. aren't those implemented with a default expression, and the column should own the backing sequence? or is that handled separately? (since tests passed i assume it's fine, but i'm just curious.)